src/LinuxCaptureService/LinuxCaptureServiceBase.cpp

```diff
@@ -149,6 +157,85 @@ class ProducerEventProcessorHijackingFunctionEntryExitForLinuxTracing
   TracingHandler* tracing_handler_;
 };
 
+absl::flat_hash_set<std::string> FileNamesOfAllMinidumps() {
+  absl::flat_hash_set<std::string> result;
+  const std::string kCoreDirectory = "/usr/local/cloudcast/core";
+  auto error_or_core_files = orbit_base::ListFilesInDirectory(kCoreDirectory);
+  if (!error_or_core_files.has_error()) {
+    const std::regex check_if_minidump_file(absl::StrFormat(R"(.*\.core\.dmp)"));
```

Comment:
Is it intended for this regex to not contain the `[0-9]+` part (number of seconds) like in `GetTargetProcessStateAfterCapture`?

Reply:
It doesn't matter. Added for consistency.